admin/app/models/certification-center-invitation.js

```diff
@@ -3,6 +3,18 @@ import Model, { attr, belongsTo } from '@ember-data/model';
 export default class CertificationCenterInvitationModel extends Model {
   @attr email;
   @attr updatedAt;
+  @attr role;
 
   @belongsTo('certificationCenter') certificationCenter;
+
+  get roleLabel() {
+    switch (this.role) {
+      case 'ADMIN':
+        return 'Administrateur';
```

Comment:
Ce serait peut être mieux d'utiliser le service d'internationalisation (pour le futur). Par contre, je ne sais pas si c'est une bonne chose d'utiliser un service dans un modèle 💭 

```suggestion
        return this.intl.t('common.roles.admin');
```

Reply:
Ce ne serait clairement pas à un modèle de l'API de faire des traductions ou de manipuler un service de traduction. Et ce serait bien aussi si on pouvait éviter ça aussi pour les modèles des apps clientes. Idéalement les traductions devraient avoir lieu uniquement dans des _components_ ou des fichiers `.hbs`.

Reply:
Je ne vois pas trop d'alternative 😕
J'ai regardé dans le code, et j'ai trouvé l'import du service `intl` dans plusieurs models côté front. Du coup, pour l'instant j'ai mis ce qu'a proposé Isma, même si je ne sais si c'est une bonne pratique.

Corrigé dans ce commit 7f2b407591398fe6b1185f935875b691af292914